Pod/Classes/Heartbeat/SEGAdobeHeartbeat.m

```diff
@@ -5,4 +5,58 @@
 //  Created by ladan nasserian on 12/2/17.
 //
 
-#import <Foundation/Foundation.h>
+#import "SEGAdobeHeartbeat.h"
+#import <Analytics/SEGAnalytics.h>
+
+@interface VideoAnalyticsProvider : NSObject <ADBMediaHeartbeatDelegate>
+
+id<ADBMediaHeartbeatDelegate> playBackDelegate();
+
+@end
+
+///-------------------------
+/// @name Media Heartbeat Initialization
+///-------------------------
+
+ADBMediaHeartbeatConfig *createConfig(SEGTrackPayload *payload, NSDictionary *settings)
+{
+    NSDictionary *app = payload.context[@"app"];
```

Comment:
the payload won't have the application context. if you do an end to end test you'll see this is always empty unless the user passes it in.

Reply:
you need to get it yourselves, you can see how we do it in the iOS library for an example.

Reply:
ok